.gitignore

```diff
@@ -2,6 +2,7 @@
 node_modules
 /.nyc_output/
 /*.iml
+.idea
```

Comment:
++

Reply:
Any IDE dependent directories or files should be ignored from the perspective of user global gitignore config not the one specific to project.

Reply:
Yikes. Sure.